<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>27</y>
     <width>602</width>
     <height>398</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView" />
      <widget class="QTabWidget" name="tabWidget" >
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>435</width>
          <height>350</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Web View</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QWebView" name="webView" >
           <property name="url" >
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>435</width>
          <height>350</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Source View</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QPlainTextEdit" name="sourceView" >
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>425</y>
     <width>602</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
